app/templates/scenario-builder.html

```diff
@@ -28,7 +28,7 @@
           </div>
 
           <!-- if user selected something, show GUI -->
-          <div>
+          <div v-if="template && dataLoaded">
```

Comment:
There is also an option for an[ async components](http://v1.vuejs.org/guide/components.html#Async-Components)  if you need to load data at component instantiation .